### PR TITLE
Added some more themes.

### PR DESCRIPTION
**Pull request by krazedkrish**

### CODE REVIEW

File: themes/crunch.bgptheme [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  GIT_PROMPT_START_ROOT="${GIT_PROMPT_START_USER}"
+  GIT_PROMPT_END_USER="${BoldBlue} ➭ ${ResetColor}"
+  GIT_PROMPT_END_ROOT="${BoldRed} # ${ResetColor}"
+  GIT_PROMPT_LEADING_SPACE=0
+  GIT_PROMPT_PREFIX="${Green}["
+  GIT_PROMPT_SUFFIX="${Green}]" 
+  GIT_PROMPT_SYMBOLS_NO_REMOTE_TRACKING="✭"
+}
+
+reload_git_prompt_colors "Solarized Archlinux"
```
Comment (magicmonty) [inline themes/crunch.bgptheme]: Could you please change the name to reflect the real name? Seems to me like a copy/paste issue ;)

Comment (magicmonty) [inline themes/crunch.bgptheme]: Could you please change the name to reflect the real name? Seems to me like a copy/paste issue ;)

File: themes/Solarized_Extravagant.bgptheme [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  GIT_PROMPT_START_ROOT="${GIT_PROMPT_START_USER}"
+  GIT_PROMPT_END_USER="\n${BoldBlue} ➭ ${ResetColor}"
+  GIT_PROMPT_END_ROOT="\n${BoldRed} # ${ResetColor}"
+  GIT_PROMPT_LEADING_SPACE=1
+  GIT_PROMPT_PREFIX="${Cyan}["
+  GIT_PROMPT_SUFFIX="${Cyan}]" 
+  GIT_PROMPT_SYMBOLS_NO_REMOTE_TRACKING="✭"
+}
+
+reload_git_prompt_colors "Solarized Archlinux"
```
Comment (magicmonty) [inline themes/Solarized_Extravagant.bgptheme]: Could you please change the theme names for crunch and "Solarized Extravagant" to reflect the real name? Seems to me like a copy/paste issue ;)

**Comment by krazedkrish** [comment]
sure :)

**Pull request closed without merging**